packages/presentation_layer/lib/app/screens/login/login_bloc/login_bloc.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:presentation_layer/app/bloc/bloc.dart';
+import 'package:presentation_layer/app/bloc/bloc_impl.dart';
+import 'package:presentation_layer/app/screens/login/login_bloc/login_data.dart';
+import 'package:domain/usecase/passwordVerify_usecase_impl.dart';
+import 'package:domain/usecase/userNameVerify_usecase_impl.dart';
+import 'package:domain/model/password_params.dart';
+import 'package:domain/model/login_params.dart';
+import 'package:presentation_layer/app/screens/login/model/login_values.dart';
+import 'package:presentation_layer/utils/constant/verify.dart' as constants;
+// import 'package:domain/model/passwordVerify_respone.dart';
+// import 'package:domain/model/userNameVerify_respone.dart';
+
+enum VerifyStatus { empty, filled, error }
+enum TypeOfInput { login, password }
```

Comment:
if u'll change onChange to set methods no need in TypeOfInput enum

Reply:
discussed